assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * SetupBanner component.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BannerNotification from '../../../../../components/notifications/BannerNotification';
+
+export default function SetupBanner( { onCTAClick } ) {
+	return (
+		<BannerNotification
+			id="ga4-activation-banner"
+			title={ __(
+				'TODO: Placeholder Setup Banner Title',
+				'google-site-kit'
+			) }
+			ctaLabel={ __( 'Connect', 'google-site-kit' ) }
+			ctaLink={ onCTAClick ? '#' : null }
+			onCTAClick={ onCTAClick }
+		></BannerNotification>
```

Comment:
Surprised this isn't an ESLint rule! 😅 

```suggestion
		/>
```